cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -34,9 +34,9 @@ pub use super::lower::isle::generated_code::MInst as Inst;
 #[derive(Clone, Debug)]
 pub struct CallInfo {
     /// Register uses of this call.
-    pub uses: SmallVec<[Reg; 8]>,
+    pub uses: SmallVec<[CallArgPair; 8]>,
     /// Register defs of this call.
-    pub defs: SmallVec<[Writable<Reg>; 8]>,
+    pub defs: SmallVec<[CallRetPair; 8]>,
```

Comment:
Added typedefs! I kept the size at `8` as IMHO it's important to avoid heap allocs for most functions here, and when I've looked before, there are a good number with more than four args. Once we have a stats framework this would be a good thing to rexamine though, I agree.